src/models/StyleSheet.js

```diff
@@ -105,6 +106,10 @@ export default class StyleSheet {
     return this.tags.map(tag => tag.toHTML()).join('')
   }
 
+  toElement() {
+    return this.tags.map(tag => tag.toElement())
```

Comment:
The elements here will need to have a `key`. You should be able to just use `cloneElement` and add it here